docs/plugin/custom-executor.md

```diff
@@ -0,0 +1,212 @@
+---
+id: custom-executor
+title: "Custom executor"
+sidebar_position: 2
+---
+
+You can extend Botkube functionality by writing custom executor plugin. An executor allows you to run a given command directly in the chat window of each communication platform. For example, running `kubectl` or `helm` commands.
+
+Plugin executor is a binary that implements the [executor](https://github.com/kubeshop/botkube/blob/main/proto/executor.proto) Protocol Buffers contract.
+
+## Goal
+
+This tutorial shows you how to build a custom `echo` executor that responds with a command that was specified by the user in a chat window.
+
+![echo-demo](assets/echo-demo.gif)
+
+## Prerequisites
+
+- Basic understanding of the Go language.
+- [Go](https://golang.org/doc/install) at least 1.18.
+- [GoReleaser](https://goreleaser.com/) at least 1.13.
```

Comment:
Same as here, maybe we can provide a link to the line we have in our gh actions

Reply:
I checked and we use `latest` 😄 won't be helpful here 😄